tkp/accessors/fitsimageblob.py

```diff
@@ -0,0 +1,53 @@
+import numpy
+import logging
+from tkp.accessors.fitsimage import FitsImage
+
+logger = logging.getLogger(__name__)
+
+
+class FitsImageBlob(FitsImage):
```

Comment:
Hm yeah maybe. I didn't want to spent too much time on this since maybe we rewrite this if we would ever switch to a more uniform Image handling structure.